xml/admin_security.xml

```diff
@@ -783,6 +783,223 @@ roleRef:
+      highly trusted users. This policy is bound to the role
+      <literal>suse:caasp:psp:privileged</literal>.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+  <para>
+   To adapt the provided example policies to your needs, copy them
+   into a YAML file. Then apply the definition by executing
+   <command>kubectl apply -f
+   <replaceable>YAML_FILE</replaceable></command>.
+  </para>
+  <para>
+   Detailed information is available at <link xlink:href=
+  "https://kubernetes.io/docs/concepts/policy/pod-security-policy/"
+  />.
+  </para>
+  <example xml:id="ex.admin.security.pod_policies.definition.unprivileged">
+   <title>Unprivileged Pod Security Policy</title>
+   <para>
+    The unprivileged Pod Security Policy is intended to be a
+    reasonable compromise between the reality of &kube; workloads and
+    the role <literal>suse:caasp:psp:privileged</literal>. By
+    default, &productname; grants this policy to all users
+    and service accounts.
+   </para>
+<screen>apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: suse.caasp.psp.unprivileged
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: docker/default
+    seccomp.security.alpha.kubernetes.io/defaultProfileName: docker/default
+    apparmor.security.beta.kubernetes.io/allowedProfileNames: runtime/default
+    apparmor.security.beta.kubernetes.io/defaultProfileName: runtime/default
+spec:
+  # Privileged
+  privileged: false
+  # Volumes and File Systems
+  volumes:
+    # Kubernetes Pseudo Volume Types
+    - configMap
+    - secret
+    - emptyDir
+    - downwardAPI
+    - projected
+    - persistentVolumeClaim
+    # Networked Storage
+    - nfs
+    - rbd
+    - cephFS
+    - glusterfs
+    - fc
+    - iscsi
+    # Cloud Volumes
+    - cinder
+    - gcePersistentDisk
+    - awsElasticBlockStore
+    - azureDisk
+    - azureFile
+    - vsphereVolume
+  allowedHostPaths:
+    # Note: We don't allow hostPath volumes above, but set this to a path we
+    # control anyway as a belt+braces protection. /dev/null may be a better
+    # option, but the implications of pointing this towards a device are
+    # unclear.
+    - pathPrefix: /opt/kubernetes-hostpath-volumes
+  readOnlyRootFilesystem: false
+  # Users and groups
+  runAsUser:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  fsGroup:
+    rule: RunAsAny
+  # Privilege Escalation
+  allowPrivilegeEscalation: false
+  defaultAllowPrivilegeEscalation: false
+  # Capabilities
+  allowedCapabilities: []
+  defaultAddCapabilities: []
+  requiredDropCapabilities: []
+  # Host namespaces
+  hostPID: false
+  hostIPC: false
+  hostNetwork: false
+  hostPorts:
+  - min: 0
+    max: 65535
+  # SELinux
+  seLinux:
+    # SELinux is unsed in CaaSP
+    rule: 'RunAsAny'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: suse:caasp:psp:unprivileged
```

Comment:
Same as above, we should change the name to something like:

    examplecompany:psp:example-custom-psp